include/vrv/editortoolkit.h

```diff
@@ -65,6 +67,8 @@ class EditorToolkit {
         int *lrx, int *lry, std::vector<std::pair<std::string, std::string>> *attributes);
     bool ParseSetAction(jsonxx::Object param, std::string *elementId, std::string *attrType, std::string *attrValue);
     bool ParseRemoveAction(jsonxx::Object param, std::string *elementId);
+    bool ParseGroupingAction(jsonxx::Object param, std::string *groupType, std::vector<std::string> *elementIds); 
```

Comment:
Should be named `ParseGroupAction` and `ParseUngroupAction` to fit with the naming scheme. Not a big deal just a style thing.